CoreTweet.Shared/Objects/Events.cs

```diff
@@ -163,7 +163,7 @@ public class TweetDeleteEvent : CoreBase
 
         [JsonProperty("timestamp_ms")]
         [JsonConverter(typeof(TimestampConverter))]
-        public DateTimeOffset Timestamp { get; set; }
+        public DateTimeOffset TimestampMs { get; set; }
```

Comment:
Thank you for fixing. But there is a precedent that `timestamp_ms` is mapped to `Timestamp` in [StreamingMessage](https://github.com/CoreTweet/CoreTweet/blob/09af87563e2187e8d4f623a4d7b9d8e30cda2f00/CoreTweet.Shared/Streaming/Messages.cs#L395-L397) because the value is represented by `DateTimeOffset` regardless of the precision of the raw value.